src/shogun/lib/memory.h

```diff
@@ -55,6 +55,10 @@ void operator delete[](void *p, std::align_val_t al);
 #else
 #define SG_ALIGNED_MALLOC(type, len, al) sg_aligned_malloc<type>(size_t(len), al)
 #endif //TRACE_MEMORY_ALLOCS
+
+#else
+#warning "Aligned malloc is not available"
```

Comment:
`#pragma message("Aligned malloc is not available")` for msvc

Reply:
right, thanks